src/components/transactions/message-view/MultiSendMessageDetails.vue

```diff
@@ -11,7 +11,7 @@
       </template>
       <template v-else-if="sentFromSessionAddress">
         <span>To&nbsp;</span>
-        <Bech32 :address="transaction.value.to_address" />
+        <Bech32 :address="transaction.value.outputs[0].address" />
```

Comment:
Hmm what happens to the other output addresses?

Reply:
We can add this fix and discuss an improvement later. Want to open an issue?

Reply:
> Hmm what happens to the other output addresses?

There are no other addresses :woman_shrugging: It works like the `inputs`, which is already implemented this way in this same file.

But yes, we can add this now and then see later. It should be fixed though, I think.

Reply:
Inputs and outputs are arrays as far as I see. Have you considered, that there could be multiple in and outputs in a multi send? Maybe the implementation before is already faulty.

Reply:
Yes, I finally realized what you meant... It is really worth investigating